src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -232,4 +232,16 @@ describe('DrawingAnnotations', () => {
             },
         );
     });
+
+    describe('state changes', () => {
+        test('should call update on the underlying popper instance when the drawnPathGroups changes', () => {
+            const popupMock = { popper: { update: jest.fn() } };
```

Comment:
Using mocks here to be consistent with how PopupReply-test.tsx handles testing these cases.